auth0-lock.js

```diff
@@ -52,6 +52,30 @@ class Auth0Lock {
     LockModule.init(this.lockOptions);
     LockModule.delegation(options, callback);
   }
+
+  /**
+   * customizeTheme - will apply the theme changes over the shared instance.
+   * @param options {obj} - theme customizations, see following link
+   *  for all possible values: https://auth0.com/docs/libraries/lock-ios/customization
+   *  - A0Theme....Color {str} - '#......' or '' for tranparent.
+   *  - A0Theme....ImageName {str}
+   *  - A0Theme....Font {obj}
+   *    - fontName {str} - ie: 'Lato-Regular'
+   *    - fontSize {number} - ie: 12.0
+   */
+  customizeTheme(options) {
+    Object.keys(options).forEach(function (key) {
+      if (key.endsWith("Color")) {
+        LockModule.registerColorForKey(options[key], key);
```

Comment:
Can you add support for android or atleast check for platform and return a warning.